modules/ucfunnelBidAdapter.js

```diff
@@ -333,7 +334,21 @@ function addUserId(bidData, userId) {
         break;
       case 'uid2':
         if (userIdObjectOrValue.id) {
-          bidData['eids'] = userIdProviderKey + ',' + userIdObjectOrValue.id
+          bidData['eids'] = (bidData['eids'].length > 0)
+            ? (bidData['eids'] + ',' + userIdProviderKey + ',' + userIdObjectOrValue.id)
+            : (userIdProviderKey + ',' + userIdObjectOrValue.id);
+        }
+        break;
+      case 'connectid':
+        if (userIdObjectOrValue) {
+          bidData['eids'] = (bidData['eids'].length > 0)
+            ? (bidData['eids'] + '!verizonMediaId,' + userIdObjectOrValue)
```

Comment:
@jackhsiehucf is `!` in `!verizonMediaId` intentional?

Reply:
yes, It is intentional, and I fixed an error in `uid2` part.
I will add `!` after bidData['eids'] when `bidData['eids'].length > 0`